### ISSUE TITLE
signin and signup are swapped in settings

### ISSUE THREAD

**Issue by Spone** [gitlabhq/gitlabhq#8968]
I just realized that on my GitLab 7.9.0.rc1 the labels for the checkboxes "Signin enabled" and "Signup enabled" are swapped in the Application settings:

```
<legend>Features</legend>
<div class="form-group">
  <div class="col-sm-offset-2 col-sm-10">
    <div class="checkbox">
      <label for="application_setting_signup_enabled">
        <input name="application_setting[signup_enabled]" value="0" type="hidden">
        <input id="application_setting_signup_enabled" name="application_setting[signup_enabled]" value="1" type="checkbox">
        Signin enabled
      </label>
    </div>
  </div>
</div>
<div class="form-group">
  <div class="col-sm-offset-2 col-sm-10">
    <div class="checkbox">
      <label for="application_setting_signin_enabled">
        <input name="application_setting[signin_enabled]" value="0" type="hidden">
        <input checked="checked" id="application_setting_signin_enabled" name="application_setting[signin_enabled]" value="1" type="checkbox">
        Signup enabled
      </label>
    </div>
</div>
```

**Comment by maxlazio** [comment]
@Spone Thanks for reporting, we will fix it for 7.9 :+1:

**Comment by Razer6** [comment]
Fixed with c31628fc001c1cf023591112a4b9e7e2ae392d03
